cmd/ltagent/init.go

```diff
@@ -64,6 +64,35 @@ func createChannels(admin *userentity.UserEntity, numChannels int) error {
 	return nil
 }
 
+func createTeamAdmins(admin *userentity.UserEntity, numUsers int, config *loadtest.Config) error {
+	for i := 0; i < numUsers; i++ {
+		index := i * 10 // making every 10th user to be a team admin
```

Comment:
Could we use something like `for i := 0; i < numUsers; i += 10 {`

Reply:
We could, but it seemed to me that this makes the intent more clear.